src/app/services/inventory.service.ts

```diff
@@ -248,18 +256,16 @@ export class InventoryService {
     }
   }
 
-  public exoticsForClass: Array<IManifestArmor> = [];
+  // I don’t think this is used?
+  // public exoticsForClass: Array<IManifestArmor> = [];
```

Comment:
Pretty sure I wanted to use this as a cache variable. Seems like I forgot to actually use it - so yes, we can probably just remove it 👍